### ISSUE TITLE
Bug: in CLI, async wait is not working

### ISSUE THREAD

**Issue by ronaldtse** [geolexica/breviter#21]
In `cli.ts`, there is this block:
```js
// TODO: Do not exit process until the save is finished!
// try {
//   const argv = await parser.parse();
// } catch (err: any) {
//   console.info(`${err.message}\n ${await parser.getHelp()}`)
// }
// console.info('finish')
parser.parse();
```

This was meant to wait for the file save process to complete before closing the process. However, this is not happening.

The goal is to uncomment this and remove the last line.

**Comment by ribose-jeffreylau** [comment]
Fixed in 2876e68f63cd664e680811699f90429d56e12588

**Comment by ronaldtse** [comment]
Awesome stuff. Thank you @ribose-jeffreylau !
